scripts/setup/packages_base_focal.lst

```diff
@@ -124,6 +123,4 @@ libsoracore-dev
 libmiro-dev
 libdecomputil-dev
 libjps3d-dev
-libopencv3.3.1
```

Comment:
shouldn't opencv4 and xfeatures2d be in here?

Reply:
added:

libopencv-core
libopencv-calib3d
libopencv-highgui
libopencv-imgcodecs
libopencv-imgproc
libopencv-features2d
libopencv-xfeatures2d
(I basically searched for all the headers in the repo and checked what was being used)